src/components/DeeplinkWrapper/index.website.js

```diff
@@ -14,10 +15,17 @@ const propTypes = {
 
 class DeeplinkWrapper extends PureComponent {
     componentDidMount() {
+        CONFIG.ENVIRONMENT = CONST.ENVIRONMENT.STAGING;
         if (!this.isMacOSWeb() || CONFIG.ENVIRONMENT === CONST.ENVIRONMENT.DEV) {
             return;
         }
 
+        const onDenyListRoute = !shouldShowDeeplink();
+
+        if (onDenyListRoute) {
+            return;
```

Comment:
I think it might need to be more complicated than this.

This stops the deeplink wrapper from _ever_ prompting if the first route loaded is a blocked one. But we actually just want to _wait_ until they load navigate to a route that isn't blocked. Otherwise, the deeplink prompt won't fire after following the desktop flow.